configuration/security.md

```diff
@@ -182,28 +187,36 @@ To disable the firewall, and disable start up on boot, use:
 sudo ufw disable
 ```
 
-Allow a particular port to have access (we have used port 22 in our example):
+What you'll want to do is deny all incoming requests by default, and allow all outgoing requests by default. This will allow you to allow only specific IP addresses to connect in on a specific port, and allow any and all IP addresses to connect out on any and all ports.
```

Comment:
Perhaps "will let you allow" reads better than "will allow you to allow" ? :wink:

Reply:
Yeah I'll change it.